docs/changes/table/14187.feature.rst

```diff
@@ -0,0 +1,2 @@
+Table and QTable can now use the ``|`` and ``|=`` operators to perform
+dictionary-style update and update in place.
```

Comment:
PEP 584 calls `|` the merge operator and `|=` the update operator. I'd prefer to be consistent with those names.
```suggestion
``Table`` and ``QTable`` can now use the ``|`` and ``|=`` operators for
dictionary-style merge and update.
```